BuildResidentialHPXML/measure.rb

```diff
@@ -1012,6 +1013,12 @@ def arguments(model)
     arg.setDefaultValue(false)
     args << arg
 
+    arg = OpenStudio::Measure::OSArgument::makeBoolArgument('heating_system_is_shared_system', true)
+    arg.setDisplayName('Heating System: Is Shared System')
+    arg.setDescription('Whether the heating system is a shared system. If true, assumed to serve all the units in the building. This is valid only for a boiler.')
+    arg.setDefaultValue(false)
+    args << arg
```

Comment:
I suggest removing this argument and changing `heating_system_type_choices` to split the boiler option into in-unit boiler vs shared boiler. Then you can automatically map IsSharedSystem in the HPXML file.